Remove references to SCA Part 2 (#2909)
theforeman/foreman-documentation: guides/common/modules/proc_generating-host-monitoring-reports.adoc

. In the {ProjectWebUI}, navigate to *Monitor* > *Reports* > *Report Templates*.

> For example, the following templates are available:
> Host {endash} Installed Products:: This generates a report with installed product information along with other metrics, including system purpose attributes.

Reviewer: ```suggestion
For example, the *Host {endash} Installed Products* template generates a report with installed product information along with other metrics, including system purpose attributes.
```
This is no longer a list, only one item is left now.

+
[NOTE]
====

> If you want to generate a subscription general report, you have to use the `Days from Now` option to specify the latest expiration time of general subscriptions.

Reviewer 2: This is referring to a specific report.

```suggestion
If you want to generate the Subscription - General report, you have to use the `Days from Now` option to specify the latest expiration time of general subscriptions.
```

> If you want to generate the Subscription - General report, you have to use the `Days from Now` option to specify the latest expiration time of general subscriptions.

Reviewer: ```suggestion
If you want to generate the *Subscription - General* report, you have to use the `Days from Now` option to specify the latest expiration time of general subscriptions.
```

> You can use the `no limit` value to show all inputs.

Reviewer 2: Subscriptions, not inputs.  (Yes, subscriptions are still around in your manifest, and that's what this report shows :)
```suggestion
You can use the `no limit` value to show all subscriptions.
```

> .Show all inputs

Reviewer 2: ```suggestion
.Show all subscriptions
```

[options="nowrap", subs="+quotes,attributes,verbatim"]
----
# hammer report-template generate \
--inputs "Days from Now=no limit" \
--name "Subscription - General Report"
----

> .Show all inputs that are going to expire within 60 days

Reviewer 2: ```suggestion
.Show all subscriptions that are going to expire within 60 days
```
